runTool.ps1

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env powershell
```

Comment:
where did you see `powershell`? I think it is called `pwsh`?

Did you test this?
If you run `./install.ps1` it will tell you if it works.